default-conventional-commits-accepted.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
those are the commits supported by the previous gh action in the platform-cicd repo